integration-tests/src/lib.rs

```diff
@@ -2,4 +2,6 @@ pub mod genesis_helpers;
 pub mod node;
 pub mod runtime_utils;
 pub mod test_helpers;
+#[cfg(test)]
+mod tests;
 pub mod user;
```

Comment:
Odd ordering of modules, I'd put `#[cfg(test)] mod tests` into a separate block separated by a blank line.

Reply:
@matklad I'm waiting for https://github.com/near/nearcore/pull/5781

Reply:
I want to move network related types to `config.rs`.